reviews/templates/mail/submitted_longroute_other_applicants.html

```diff
@@ -0,0 +1,35 @@
+{% extends "mail/base.html" %}
+
+{% block content_nl %}
+{% if was_revised %}
+Beste collega,<br/>
+<br/>
+Uw collega {{ creator }} heeft een gereviseerde aanvraag ingediend voor een onderzoek, waar u aan meewerkt, bij de ethische commissie. De aanvraag is door de FETC-GW ontvangen.<br/>
+<br/>
+Dit houdt in dat de revisie nu door twee leden van de FETC-GW zal worden beoordeeld; een eenvoudige revisie wordt alleen door de secretaris beoordeeld. We hopen uw collega, {{ creator }}, binnenkort een beoordeling te kunnen sturen.<br/>
```

Comment:
The supervisor email has a link to the PDF of the proposal. It might be nice to add it here as well

(and in the shortroute email as well)